pkg/extensions/lint/lint.go

```diff
@@ -123,3 +140,38 @@ func getMissingAnnotations(mandatoryAnnotationsMap map[string]bool) []string {
 
 	return missingAnnotations
 }
+
+func (linter *Linter) checkIfLogoAnnotationValid(annotation string, mandatoryAnnotations map[string]string) bool {
+	if val, ok := mandatoryAnnotations[logoKey]; ok {
```

Comment:
If we pass the annotation value as parameter we should not need this if. It should be cleaner. This function should be responsible for validating annotations, and it shouldn't need to be aware that some of them are mandatory and some ore not.

Reply:
Changed the signature and it's usages.